### PR TITLE
Final edits before submitting to a vote

### PR DESCRIPTION
**Pull request by bsdimp**

### CODE REVIEW

File: fcp-0100.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ unnecessary friction.
 
 While it initially felt like similar to the difference between i486
 and i586, the number of changes required to make this happen are
-larger, especially in ports. Linux has agressively adopted armv7 for
-it's configuration, so having our uname -p match Linux will make the
-ports easier to adopt.
+larger, especially in ports. Linux has agressive adopted armv7 in its
```
Comment (allanjude) [inline fcp-0100.md]: I think you actually meant 'agressively', or 'been agressive in'

Comment (allanjude) [reply fcp-0100.md]: s/in its/as its/ ?

File: fcp-0100.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ unnecessary friction.
...
+processor name. In addition, being able to optimize different between
```
Comment (allanjude) [inline fcp-0100.md]: s/different/differently/

**Review by allanjude — CHANGES_REQUESTED** [review]
Few fixups needed

File: fcp-0100.md (hunk 1 above)
Comment (bsdimp) [reply fcp-0100.md]: yup fixed.

File: fcp-0100.md (hunk 2 above)
Comment (bsdimp) [reply fcp-0100.md]: Fixed.

**Review by allanjude — APPROVED** [review]

**Pull request merged**